content/articles/introduction-to-ensemble-methods-in-machine-learning/index.md

```diff
@@ -0,0 +1,96 @@
+# Introduction to Ensemble Methods in Machine Learning
+
+## Prerequisites

```

Comment:
As mentioned in the guidelines (also in the PR checklist), the markdown headers must start from `###`, not `##`

Reply:
Bring pre-requisites section after table of contents